pkg/kubelet/pluginmanager/plugin_manager.go

```diff
@@ -68,11 +69,16 @@ func NewPluginManager(
 		asw,
 	)
 
+	// We rate limit registering new plugins to prevent denial of service.
+	var pluginRegistrationQPS float32 = 10.0
```

Comment:
These should be consts instead of vars. Also I'm not sure if these are what to want to set in the code, or users should be able to set this in the config 🤔 @saad-ali what's your opinion on this?